src/gtp/mod.rs

```diff
@@ -204,14 +205,14 @@ impl<'a> GTPInterpreter<'a> {
                 }
 
             },
-            "loadsgf" => {
+            Some(knownCommands::loadsgf) => {
                 let filename = command[1];
                 let parser = Parser::from_path(Path::new(filename));
                 let game = parser.game().unwrap();
                 self.game = game;
                 Command::LoadSgf
             },
-            _             => return Command::Error
+            None             => return Command::Error
```

Comment:
Awesome! So I assume that means that the `Some` cases have to be exhaustive, and the compiler will throw an error if one is missing? That would be brilliant.